src/pages/Settings/Settings.jsx

```diff
@@ -0,0 +1,110 @@
+import { useContext, useState } from 'react';
+import Toogle from '../../ui-kit/components/Toogle';
+import Avatar from '../../ui-kit/components/Avatar';
+import {
+  AvatarList,
+  ChangeInputStyled,
+  NewSettingsWrap,
+  SaveChangeButton,
+  SettingsIcon,
+  SettingsIconButton,
+  SettingsIconEdit,
+  SettingsLabel,
+  SettingsPageMainTitle,
+  SettingsPageWarp,
+  SettingsWrap,
+} from './Settings.styled';
+import { ThemeContext } from '../../ui-kit/theme/ThemeProvider';
+import { ListItem } from '@mui/material';
+
+const SettingsPage = () => {
+  const { toggleTheme, currentTheme } = useContext(ThemeContext);
+
+  const [isChangeNameVisible, setIsNameChangeVisible] = useState(false);
+  const [isChangeAvatarVisible, setIsAvatarChangeVisible] = useState(false);
+
+  const handleChangeNameClick = () => {
+    setIsNameChangeVisible(!isChangeNameVisible);
+  };
+
+  const handleChangeAvatarClick = () => {
+    setIsAvatarChangeVisible(!isChangeAvatarVisible);
+  };
+
+  return (
```

Comment:
Each page renders only its own component, move everything to the components/SettingsPageCopmonent folder